### PR TITLE
Adding NewRelic coverage to discord

### PR DESCRIPTION
**Pull request by darkwing**
Sooooooo...I'm not sure exactly how to test this outside of pushing.  I've followed the setup directions, as instructed.  I've set license key and app name environment variables on production.

### CODE REVIEW

**Comment by darkwing** [comment]
Closing this, I don't believe we need it.  Bear with me :)

**Comment by darkwing** [comment]
Reopening as the docs say we should use this:  https://devcenter.heroku.com/articles/newrelic#node-js-installation-and-configuration

**Comment by darkwing** [comment]
OK, so here's what I've done:

1.  Installed newrelic on prod: `heroku addons:create newrelic:wayne`
2.  Overrode the default name and keys by setting explicit `heroku config:set NEW_RELIC_LICENSE_KEY={key}` and `heroku config:set NEW_RELIC_APP_NAME=MDNDiscord`
3.  Added the new relic resources to the package.

I believe this is all that's required.

File: newrelic.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  license_key: process.env.NEW_RELIC_LICENSE_KEY, // Overriden by environment vars
+  logging: {
+    /**
+     * Level at which to log. 'trace' is most useful to New Relic when diagnosing
+     * issues with the agent, 'info' and higher will impose the least overhead on
+     * production applications.
+     */
+    level: 'info'
+  }
+}
```
Comment (openjck) [inline newrelic.js]: We can add a semicolon here to pass JSHint

File: app.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+require('newrelic');
```
Comment (openjck) [inline app.js]: I notice this isn't mentioned in the Heroku documentation. I don't doubt that it's the right thing to do, but just to be sure I understand would you be able to say more?

File: newrelic.js (hunk 1 above)
Comment (openjck) [inline newrelic.js]: We could also run `gulp beautify` to keep consistent coding conventions in this file.

Comment (openjck) [inline newrelic.js]: Does New Relic have documentation on this file? I'm wondering how it works&mdashdoes the `newrelic` module look for a global object called `config` or does this file play a role as a piece in a bigger puzzle?

**Pull request merged**